dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/ErrorType.java

```diff
@@ -43,7 +43,7 @@
    * not public. If the source class is in the same package, the class or one of its enclosing types
    * is private.
    */
-  INACCESSIBLE_CLASS,
+  INACCESSIBLE_CLASS("is not accessible class"),
```

Comment:
I don't think you need "class: at the the end, but if you do then "is not an accessible class"

Reply:
Removed "class" at the end.